apps/server/src/modules/files-storage/controller/files-storage.controller.ts

```diff
@@ -81,4 +82,38 @@ export class FilesStorageController {
 
 		return response;
 	}
+
+	@Get('/delete/:schoolId/:parentType/:parentId')
```

Comment:
Why not @Delete('/by-parent/:schoolId/:parentType/:parentId')

Reply:
delete/parent/

Reply:
It is still open for the moment we have:
'/delete/:schoolId/:parentType/:parentId'
'/delete/:fileRecordId'